satellite/audit/reservoir.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+	"storj.io/storj/pkg/pb"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Segments []*pb.RemoteSegment
+	IsVetted bool
+	NumSlots int
```

Comment:
How about just `Slots`?

When referenced it would look like `Reservoir.Slots` which reads well.

Reply:
Or maybe we should call it `Size`.

Reply:
okay I think I also prefer Size because Slots to me sounds like it would be a map or array